packages/core/src/components/slider/rangeSlider.tsx

```diff
@@ -44,6 +45,9 @@ export interface IRangeSliderProps extends ISliderBaseProps {
 
     /** Callback invoked when a handle is released. */
     onRelease?(value: NumberRange): void;
+
+    /** HTML props to apply to the slider Handles */
+    handleHtmlProps?: { start: HandleHtmlProps; end: HandleHtmlProps };
```

Comment:
I just realized that these should probably be optional, so you can only set one of them if you want:

```suggestion
    handleHtmlProps?: { start?: HandleHtmlProps; end?: HandleHtmlProps };
```